utils/build/docker/golang/app/gqlgen/server.go

```diff
@@ -55,7 +60,21 @@ func main() {
 		w.Write(jsonData)
 	})
 
+	httpSrv := &http.Server{
+		Addr:    ":7777",
+		Handler: mux,
+	}
+
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, syscall.SIGTERM)
+	go func() {
+		for range c {
+			httpSrv.Shutdown(context.Background())
+			return
+		}
+	}()
+
 	common.InitDatadog()
 
-	panic(http.ListenAndServe(":7777", mux))
+	panic(httpSrv.ListenAndServe())
```

Comment:
I think this will always panic now that you actually stop the server, since `ListenAndServer` returns `http.ErrServerClosed` when you call `Shutdown`.